server/src/test/java/org/opensearch/index/mapper/NumberFieldTypeTests.java

```diff
@@ -981,4 +982,27 @@ public void testBitmapQuery() throws IOException {
         NumberFieldType finalFt = ft;
         assertThrows(IllegalArgumentException.class, () -> finalFt.bitmapQuery(bitmap));
     }
+
+    public void testFetchUnsignedLongDocValues() throws IOException {
+        Directory dir = newDirectory();
+        IndexWriter w = new IndexWriter(dir, new IndexWriterConfig(null));
+        Document doc = new Document();
+        doc.add(new SortedNumericDocValuesField("ul", 10));
```

Comment:
Could you please use random value (needs a bit of adjustment probably):

```suggestion
        doc.add(new SortedNumericDocValuesField("ul", randomUnsignedLong()));
```

Reply:
done